scenarios/kubernetes_bazel.py

```diff
@@ -71,10 +71,15 @@ def main(args):
             res = 1
         else:
             try:
-                check(
-                    'bazel', 'run', '//:push-build', '--',
-                    '%s/%s' % (args.gcs, version)
-                    )
+                gcs_location = '%s/%s' % (args.gcs, version)
+                check('bazel', 'run', '//:push-build', '--', gcs_location)
```

Comment:
I do think the push-build could probably do this (@ixdy?) but I'm not sure if it *should* (?)